engine/legacy-charts/src/main/java/org/pentaho/plugin/jfreereport/reportcharts/ColorHelper.java

```diff
@@ -80,7 +81,13 @@ private ColorHelper() {
     colors.put( "gold", Color.decode( "#ffd700" ) ); //$NON-NLS-1$ //$NON-NLS-2$
     colors.put( "goldenrod", Color.decode( "#daa520" ) ); //$NON-NLS-1$ //$NON-NLS-2$
     colors.put( "gray", Color.decode( "#808080" ) ); //$NON-NLS-1$ //$NON-NLS-2$
-    colors.put( "green", Color.decode( "#008000" ) ); //$NON-NLS-1$ //$NON-NLS-2$
+    if ( !ClassicEngineBoot.getInstance().getExtendedConfig().
+      getBoolProperty( "org.pentaho.reporting.engine.classic.extensions.legacy.charts.LegacyColorMode" ) ) {
+      colors.put( "green", Color.decode( "#00ff00" ) ); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
So there is no "lime" color defined when legacy mode is off? What color will reports see when they previously defined a "lime" bar chart?